Formula/mbt.rb

```diff
@@ -0,0 +1,33 @@
+class Mbt < Formula
+  desc "Multi-Target Application (MTA) build tool for Cloud Applications"
+  homepage "https://sap.github.io/cloud-mta-build-tool"
+  url "https://github.com/SAP/cloud-mta-build-tool/archive/v1.2.10.tar.gz"
+  sha256 "46d548342e2fe3abd51d4c5d5702602b835a2cfd53eb09b86a2abf45825823a4"
+  license "Apache-2.0"
+  head "https://github.com/SAP/cloud-mta-build-tool.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    target = Hardware::CPU.intel? ? "build-darwin" : "build-darwin-arm"
+
+    if Hardware::CPU.intel?
+      target = "build-darwin"
+      binary = "mbt"
+    else
+      target = "build-darwin-arm"
+      binary = "mbt_darwin_arm"
+    end
+
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/SAP/cloud-mta-build-tool").install buildpath.children
+    cd "src/github.com/SAP/cloud-mta-build-tool" do
+      system "make", target
+      bin.install "release/#{binary}" => "mbt"
+    end
```

Comment:
Doesn't this work like this?
```suggestion
    system "go", "build", *std_go_args, "./src/github.com/SAP/cloud-mta-build-tool/"
```

Reply:
@SMillerDev thank you, and yes, it's much simpler by just doing the essentially steps as in the Makefile. And now the build has passed on all platforms, cheers!